spec/griddler/cloudmailin/adapter_spec.rb

```diff
@@ -16,7 +16,8 @@
       cc: ['emily@example.com'],
       from: 'Joe User <joeuser@example.com>',
       subject: 'Re: [ThisApp] That thing',
-      text: /Dear bob/
+      text: /Dear bob/,
+      headers: { Subject: 'Re: [ThisApp] That thing', Cc: 'emily@example.com' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.